system-tests/nodes/test-nodes.js

```diff
@@ -0,0 +1,21 @@
+const hostnameSelector =
+  ".BottomLeftGrid_ScrollWrapper .ReactVirtualized__Grid__innerScrollContainer a";
+
+describe("Nodes", function() {
+  afterEach(() => {
+    cy.window().then(win => {
+      win.location.href = "about:blank";
+    });
+  });
+
+  it("shows all nodes as healthy", () => {
+    cy.visitUrl("/nodes");
+    cy.get(hostnameSelector).should("have.length", 2);
+  });
+
+  it("shows all nodes as healthy", () => {
+    cy.get(
+      ".ReactVirtualized__Grid__innerScrollContainer > div:contains('Healthy')"
```

Comment:
Why not extracting this as a const as well? Shall we write some cypress helpers for the new table?

Reply:
Would rather make the new table a bit more approachable with a few extra classes in the future.